public/class-admwswp-public.php

```diff
@@ -149,6 +149,11 @@ public static function weblinkWidget($attr)
                     $webLinkArgs['id'] = $path_id;
                 }
                 break;
+            case 'PathObjectives':
```

Comment:
can this be combined with `case 'PathDetails':`

Reply:
no it has different arguments